config/controller/kustomization.yaml

```diff
@@ -6,4 +6,4 @@ kind: Kustomization
 images:
 - name: controller
   newName: public.ecr.aws/aws-controllers-k8s/sqs-controller
-  newTag: 1.0.3
+  newTag: 1.0.2
```

Comment:
Actually, @A-Hilaly you will need to rebase and correct this. Should not be going backwards by a version. :)